### PR TITLE
validator(dm): unify command name & more integration test

### LINKED ISSUE #5632: add `allProcessedRowStatus` into validation status, so we can check validator works in infra test on failover or restart from checkpoint, or later.

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #5368 
close #5632 

### What is changed and how it works?
- unify dmctl command name related to error change to xxx-error
- `processedRowsStatus` in `validation status` means all processed row since validation enabled. also rename `allErrorRowsStatus` to `errorRowsStatus`
- add more integration test
- add more debug logs;

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Comment by D3Hunter** [comment]
ptal @lance6716 @buchuitoudegou @Ehco1996

**Review by buchuitoudegou — COMMENTED** [review]
lgtm (except IT, will take a look at it in a second)

File: dm/tests/dmctl_command/run.sh [hunk 1]
```diff
@@ -562,6 +589,40 @@ function run_validator_cmd {
...
+		"validation ignore-error test --all"
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-error test" \
+		"\"id\": " 0
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation status test" \
+		"new\/ignored\/resolved: 0\/4\/0" 1
 }
 
 function run_validator_cmd_error() {
```
Comment (buchuitoudegou) [inline dm/tests/dmctl_command/run.sh:628]: What about adding some failed cases of inexistent task names or error ids? The former case is supposed to print the `not found` message while the latter one is supposed to succeed quietly.

**Review by buchuitoudegou — COMMENTED** [review]
rest LGTM

File: dm/dm/ctl/master/operate_validation.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ import (
 func NewIgnoreValidationErrorCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "ignore-error <task-name> <error-id|--all>",
```
[the remark this answers is not in the document]
Comment (D3Hunter) [reply dm/dm/ctl/master/operate_validation.go:30]: handle-error is a command, this is a sub-command, full command is `dmctl validation ignore-error`

File: dm/tests/dmctl_command/run.sh (hunk 1 above)
Comment (D3Hunter) [reply dm/tests/dmctl_command/run.sh:628]: for non-exist error id, we should return a message, not success silently.

add a task in https://github.com/pingcap/tiflow/issues/4426. i t for this case should be added when we fix it.

Comment (D3Hunter) [reply dm/tests/dmctl_command/run.sh:628]: [add invalid task name case](https://github.com/pingcap/tiflow/pull/5633/commits/157554b914af3736a890ea44bfa7067bbc72bb89)

**Comment by D3Hunter** [comment]
ptal @lance6716 @buchuitoudegou

**Review by buchuitoudegou — APPROVED** [review]
lgtm

File: dm/tests/dmctl_command/run.sh [hunk 3]
```diff
@@ -431,7 +433,15 @@ function run_validation_start_stop_cmd {
 		"\"stage\": \"Stopped\"" 1
 }
 
-function run_validator_cmd {
+function trigger_checkpoint_flush() {
+	sleep 1.5
```
Comment (Ehco1996) [inline dm/tests/dmctl_command/run.sh:437]: it seems that sleep after run_sql is better?

File: dm/tests/validator_basic/run.sh [hunk 4]
```diff
@@ -34,6 +35,7 @@ function prepare_for_standalone_test() {
 }
 
 function trigger_checkpoint_flush() {
+	sleep 0.1
```
Comment (Ehco1996) [inline dm/tests/validator_basic/run.sh:38]: ditto, and why here only need to sleep 0.1 s?

**Review by Ehco1996 — COMMENTED** [review]
rest LGTM

**Review by Ehco1996 — APPROVED** [review]
LGTM

**Pull request merged**